frontend/packages/dev-console/src/components/import/GitImport.tsx

```diff
@@ -72,26 +66,17 @@ const GitImport: React.FC<GitImportProps> = ({ namespace, imageStreams }) => {
 
     const {
       project: { name: projectName },
-      route: { create: canCreateRoute },
-      image: { ports },
     } = values;
 
-    const requests = [
-      createDeploymentConfig(values, imageStream),
-      createImageStream(values, imageStream),
-      createBuildConfig(values, imageStream),
-    ];
+    const dryRunRequests: Promise<K8sResourceKind>[] = createResources(values, imageStream, {
+      queryParams: { dryRun: 'All' },
```

Comment:
Simplify this by just sending `dryRun=true` as parameter. `createResources` function or maybe specific function that create resource should figure out what parameter it needs to send to `k8sCreate`.

Reply:
Done